cirq-google/cirq_google/workflow/quantum_runtime.py

```diff
@@ -50,14 +51,18 @@ class SharedRuntimeInfo:
 
     run_id: str
     device: Optional[cirq.Device] = None
+    run_start_time: Optional[datetime.datetime] = None
+    run_end_time: Optional[datetime.datetime] = None
 
     @classmethod
     def _json_namespace_(cls) -> str:
         return 'cirq.google'
 
     def _json_dict_(self) -> Dict[str, Any]:
+        d = dataclass_json_dict(self)
         # TODO (gh-4699): serialize `device` as well once SerializableDevice is serializable.
```

Comment:
LOL

Reply:
they changed my original issue title to be a _little_ less ridiculous https://github.com/quantumlib/cirq/issues/4699